src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -186,6 +287,9 @@ private static void MoveRuleDescriptors(JObject run)
             // 1. 'run.resources.messageStrings' moves to 'run.tool.globalMessageStrings'
             if (resources["messageStrings"] is JObject messageStrings)
```

Comment:
>messageStrings [](start = 27, length = 14)

Wow, that was an oversight. The `resources` object never had `richMessageStrings`.